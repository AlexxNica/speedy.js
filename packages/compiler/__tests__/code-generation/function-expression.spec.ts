import {runCases} from "./code-generation-test-case-runner";

runCases("FunctionExpression", "function-expression");
