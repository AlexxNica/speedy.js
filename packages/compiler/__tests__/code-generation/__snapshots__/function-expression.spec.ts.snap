// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FunctionExpression function-expression-error 1`] = `
"__tests__/code-generation/cases/function-expression/function-expression-error.ts(4,16): error TS100005: The type '(x: number, y: number) => number' is not supported.
"
`;
